### ISSUE TITLE
Black patches on curve in Vega

### ISSUE THREAD

**Issue by vloup** [DaemonEngine/Daemon#796]
I have observed some black tris on the map Vega exclusively, using the nvidia driver (currently running 525.89.02 on a GTX 970).
I don't see the problem with my old Intel HD 4000.
I've already given a glxinfo in the past: #472

Location:
```
]/viewpos  
1026 164 381 43 28
```

I cannot tell you exactly which version this problem started with. It's been there for a long time. I was just procrastinating for reporting it.

I've tested the different rendering presets:
- lowest: no black patches
- low: no black patches
- medium: single black patches
- high: multiple black patches
- ultra: multiple black patches

I guess the change of the number of black patches might be related to the LoD option.

Medium:
![unvanquished_2023-02-12_145850_000](https://user-images.githubusercontent.com/2750613/218315857-a7d55475-60ed-4601-b9f0-01de88d95a0c.jpg)

High:
![unvanquished_2023-02-12_145905_000](https://user-images.githubusercontent.com/2750613/218315858-3967d26e-5961-407d-973c-006b5d65a886.jpg)

Ultra:
![unvanquished_2023-02-12_145926_000](https://user-images.githubusercontent.com/2750613/218315859-809057f8-8a59-4804-85e4-0dc2b2d09de0.jpg)

**Comment by illwieckz** [comment]
To verify it's related to LoD, one may try different values of `r_subdivisions`, it's probably needed to call `vid_restart` after changing LoD.

**Comment by vloup** [comment]
Good idea. Tested in Ultra, with r_showtris 1. It is surprisingly not aligned with the edges.

I forgot to mention:
```
]/version  
"version" - "Unvanquished 0.54.0 Linux amd64 Jan 30 2023" - default: "Unvanquished 0.54.0 Linux amd64 Jan 30 2023"
```

r_subdivisions 1:
![unvanquished_2023-02-12_153243_000](https://user-images.githubusercontent.com/2750613/218317260-d7b6813a-fd81-406b-91a8-187e3f1dc374.jpg)

r_subdivisions 4:
![unvanquished_2023-02-12_153259_000](https://user-images.githubusercontent.com/2750613/218317262-4ad5de84-974e-41af-885f-0cbc9f32ba39.jpg)

r_subdivisions 8:
![unvanquished_2023-02-12_153340_000](https://user-images.githubusercontent.com/2750613/218317263-d06b5a60-1b35-46e4-a0e8-aefb16add58e.jpg)

**Comment by vloup** [comment]
As of 04-01-2024, with driver version 545.29.06 on unv 0.54.1, the patches are not there anymore (tested in Ultra graphics mode).

![unvanquished_2024-01-04_154003_000](https://github.com/DaemonEngine/Daemon/assets/2750613/cc6b4a02-8bf7-4b73-86ac-707f95da4e30)
